specs/App.spec.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { render } from '@testing-library/react-native';
+
+import App from '../App';
+
+describe('<App />', () => {
+  it('has 1 child', () => {
+    const screen = render(<App />)
+    expect(screen.getByText('TuneShare').children).toHaveLength(1)
```

Comment:
What is this testing?🤔 Whether <Text>TuneShare</Text> has children or not???

Reply:
it's just a test to see if jest is working. after we create our first real test we will delete this one.

Reply:
Yeah- until everyone's personal repos are up and running, it's a good barometer to see that whether the dependencies have installed correctly

Reply:
Oh that's a good idea! I'm convinced👍